reports/default/examples/empty.json

```diff
@@ -1262,8 +1262,12 @@
                     "currency": {
                         "type": null
                     },
+                    "mileage": null,
                     "related_ads": [
                         {
+                            "date": {
+                                "publish": "2019-01-14 11:57:00"
```

Comment:
Change `"publish": null`, for this case.

Reply:
Fixed